src/app/shared/user/user.query.ts

```diff
@@ -11,6 +11,7 @@ export class UserQuery extends Query<UserState> {
   user$: Observable<User> = this.select(state => state.user);
   isCurator$: Observable<boolean> = this.select(state => (state.user ? state.user.curator : null));
   isAdmin$: Observable<boolean> = this.select(state => (state.user ? state.user.isAdmin : null));
+  isAdminOrCurator$: Observable<boolean> = this.select(state => (state.user ? state.user.curator || state.user.isAdmin : null));
```

Comment:
```
  isAdminOrCurator$: Observable<boolean> = combineLatest(this.isAdmin$, this.isCurator$).pipe(
    map(([isAdmin, isCurator]: [boolean, boolean]) => {
      return isAdmin || isCurator;
    })
  );
```
is one way of doing it without getting the state and avoiding null-checking the user again.  Yours is ok too